src/main/java/FindRepetitiveChar/ConstainSolution.java

```diff
@@ -0,0 +1,21 @@
+package FindRepetitiveChar;
+
+public class ConstainSolution {
+    public static void main(String[] args) {
+        String characters="ALKSSACD";
+        System.out.println(findRepetitiveCharWithContains(characters));
+    }
+
+    private static char findRepetitiveCharWithContains(String characters) {
+        String newcharacters;
+        for (int i = 0; i < characters.length(); i++) {
+            newcharacters = characters.substring(i+1, characters.length());
+            if (newcharacters.contains(characters.charAt(i) + "")) {

```

Comment:
```suggestion
char firstChar;
   for (int i = 0; i < characters.length(); i++) {
          newcharacters = characters.charAt(i);
            if (characters.substring(i+1, characters.length()).contains(firstChar)) {
```
Bunu dener misin? Ne dusunuyorsun bunun hakkinda?